api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -38,6 +38,17 @@ message ExtAuthz {
   // Note that errors can be *always* tracked in the :ref:`stats
   // <config_http_filters_ext_authz_stats>`.
   bool failure_mode_allow = 2;
+
+  // Enables filter to buffer and add the request body to the authorization request.
+  BufferSettings with_request_body = 4;
```

Comment:
Add a note on how this play with `failure_mode_allow`? seems with current impl, if request body exceeds the buffer setting it will result 413 regardless of `failure_mode_allow`.